includes/Stories_Renderer/Renderer.php

```diff
@@ -528,7 +528,12 @@ protected function render_story_with_poster() {
 				>
 				</amp-img>
 			<?php } else { ?>
-				<img src="<?php echo esc_url( $poster_url ); ?>" alt="<?php echo esc_attr( $story_data->get_title() ); ?>">
+				<img
+					src="<?php echo esc_url( $poster_url ); ?>"
+					alt="<?php echo esc_attr( $story_data->get_title() ); ?>"
+					width="<?php echo absint( $this->width ); ?>"
+					height="<?php echo absint( $this->height ); ?>"
+				>
```

Comment:
```suggestion
				printf( '<img src="%s" alt="%s" width="%d" height="%d" />', esc_url( $poster_url ), esc_attr( $story_data->get_title() ), absint( $this->width ), absint( $this->height ) );
```

How about something like this?

Reply:
Also, what about lazy loaded images here?

Reply:
Then we'd also have to do it for `amp-img` above for consistency. But this feels way less readable to me.

Reply:
>  But this feels way less readable to me.

Feels more readable to me.